README.md

```diff
@@ -192,6 +196,25 @@ You have two options to run the site locally after the first install:
 
 **Note**: changes that alter site navigation or other changes that change the site as a whole might not show up when using `--incremental`. If that occurs, simply “kill” the build and run `bundle exec jekyll serve` without the suffix. **This is also true for gulp: you will need to kill your gulp instance and then run the direct Jekyll command**.
 
+## Hiding files
+
+* Add the files to the [Hidden/Hidden folder](https://github.com/LivePersonInc/developers-community/tree/Staging/pages/Documents/Hidden/Hidden).
+* Do not include them in the documentsupdated.yaml file.
+* Ensure the `layout` header is set to `hidden-layout` so that search engines do not find it.
+* Follow the header structure of [`blank.md`](https://github.com/LivePersonInc/developers-community/blob/Staging/pages/Documents/Hidden/Hidden/blank.md) in the Hidden/Hidden folder to set the headers correctly.
+
+**Any other parameters not documented here, but in the front matter of other files, are deprecated and only present for backwards-compatibility. They should not be used.**
+
+### Adding new documents to the sidebar
+
+Once you’ve created a new document, you’ll need to have it manually added. We chose a manual process for the sidebar for a few reasons:
+
+1. First, it reduces the fragility of the sidebar (the extra, manual step gives us another layer of QA).
```

Comment:
I think there is no need here for First, Second, etc. because this is implied in the numbers.

Reply:
Updated.